.github/ISSUE_TEMPLATE/TASK.yaml

```
@@ -1,11 +1,75 @@
----
-name: 🏃🏻‍♂️ Task
-description: A general task that does not fit into other categories.
-title: "[task]: "
-type: Task
+name: "Task"
+description: "Work item that goes through refinement before development"
```

I like something like this more:

```suggestion
description: "A general task with a clear purpose."
```

I do not feel how it is used by DVT is relevant. Others might find it useful without following our refinement process.